requirements.txt

```diff
@@ -46,3 +46,4 @@ tabulate
 torchvision==0.5.0
 tqdm
 h5py
+pretrainedmodels
```

Comment:
Are you using `se_resnext50_32x4d`? This dependency is only need for this network.
If you really need this dependency i would recommend adding the version number you tested to the requirements.txt. 
This is not an official site for pretrained models and I would be worried about using the latest releases without testing first.

Reply:
I am using it, if possible I would like to keep the dependency. It is not an official site, but it is a widely used repository for pretrained models, it has more stars in github than torchvision itself. Sure, I can add the specific version number to requirements.

Reply:
I am using. Despite not being official, it is a widely used library, has more stars than github - the large amount of pretrained models will be useful for us. It is also very lightweight.  

There is an open PR in pretrainedmodels to add support to load these models through torchhub, in the next release we might not need to import it directly.

I can add the versioning in requirements